client/src/pages/CalendarPage/CalendarPage.js

```diff
@@ -31,24 +33,55 @@ const dailyTaskDateFormatter = (date) => {
   } ${date.getDate()}, ${date.getFullYear()}`;
 };
 
+// Convert the task data fetched from the server into JS objects of the
+// {title, locationText, startTimeText, endTimeText, isOngoing}
+const convertTaskData = (fetchedTaskData) => {
+  return [];
+};
+
+const padSingleDigit = (num) => {
+  return num <= 9 ? `0${num}` : num.toString();
+};
+
+const buildDailyTaskRoute = (currDate) => {
+  // getMonth returns 0-11
+  const currMonthStr = padSingleDigit(currDate.getMonth() + 1);
```

Comment:
Having something like this also works as well instead of having another function to do the padding
https://stackoverflow.com/questions/10073699/pad-a-number-with-leading-zeros-in-javascript
![image](https://user-images.githubusercontent.com/24628243/160270248-b0e7aa9c-68c2-4091-bdb5-9377a9633039.png)